package.json

```diff
@@ -173,9 +172,10 @@
   },
   "dependencies": {
     "@apollo/client": "^3.2.0",
-    "@neo4j/browser-lambda-parser": "1.0.5",
+    "@juggle/resize-observer": "^3.3.1",
```

Comment:
can we remove this all together since all browsers that we support now don't need this?

Reply:
I removed this but then noticed our typescript version didn't have types for the resize-observer yet, started upgrading it which meant I had to fix new type errors, install a new version of ts-jest and so on. Turned out to be too much of a rabbit hole for now, so I'd prefer to keep the polyfill in for now :+1: